platform/darwin/src/MGLNetworkConfiguration.mm

```diff
@@ -66,6 +67,19 @@ - (NSURLSessionConfiguration *)defaultSessionConfiguration {
     return sessionConfiguration;
 }
 
+- (void)setStopsRequests:(BOOL)stopsRequests {
```

Comment:
`stopRequests` is not really a great name for this API, because is a double negation.

Reply:
I agree with this; @tobrun will `setConnected` remain the same? 

@fabian-guerra how about we go with a `connected` property? My concern here is that this might be confused whether there is a valid connection (a la reachability).